release/release-notes/data-prepper.release-notes-2.0.0.md

```diff
@@ -0,0 +1,53 @@
+## 2022-10-23 Version 2.0.0
+
+---
+
+### Features
+* Support conditional routing in sinks ([#1337](https://github.com/opensearch-project/data-prepper/issues/1337))
+* Support Core Peer Forwarder which replaces the peer forwarder processor plugin ([#700](https://github.com/opensearch-project/data-prepper/issues/700))
+* Support parsing CSV and TSV values in event fields using CSV processor ([#1081](https://github.com/opensearch-project/data-prepper/issues/1081))
```

Comment:
We also support directly decoding an S3 object as CSV which is a nice new feature. We should include that.